site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
+| Use zip/video chunks | Use this parameter to speed up the annotation process for videos or image collections. When enabled, the server will divide <br>the video into smaller chunks and send them to the client side in an archive, making it easier to work with the data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
The definition is wrong. The whole idea of the option is in the line https://github.com/opencv/cvat/blob/be8444581766eff60bff560fcd8739e95307cdc6/cvat/apps/engine/task.py#L523

Video chunks can be prepared as short video clips or as zip of several frames. I'm not sure that we really need these two options. But zip of frames are bigger in size but you don't need to decode them on the client machine (just unzip). Short video clips are smaller, but the client machine should decode them in browser.

I will recommend turning off the option of video tasks. It will reduce the amount of traffic between the client and the server. Try to experiment with these two options and see if you have any difference on your machine.